Add "system_settings" config doc (#74)
harvester/docs: docs/install/harvester-configuration.md

  vip_mode: dhcp
  vip_hw_addr: 52:54:00:ec:0e:0b
```

### `system_settings`

#### Definition

> Use this config to overwrite default Harvester system settings.

Author: Rephrased

!!! note

>     Overwriting system settings only works when Harvester is installing in "create" mode. When installing in "join" mode, this config has no effect.

Author: Rephrased

#### Example

> Automatically adding storage devices on every Nodes that match the glob pattern as VM storages.

Reviewer: ```suggestion
The configuration below will automatically scan and add storage devices which match the glob pattern `/dev/sd*` on every Node.
```

Author: Rephrased

  vip_mode: dhcp
  vip_hw_addr: 52:54:00:ec:0e:0b
```

### `system_settings`

#### Definition

> This setting allows you to overwrite the default Harvester system settings.

Reviewer: ```suggestion
You can overwrite the default Harvester system settings by configuring `system_settings`.
```

!!! note

>     Overwriting system settings is only applicable if Harvester is installed in "create" mode.
>     Instead, if the "join" mode is selected during the Harvester installation, this setting will
>     be discarded.

Reviewer: ```suggestion
    Overwriting system settings only works when Harvester is installed in "create" mode.
    If you install Harvester in "join" mode, this setting is ignored.
```

Author: In addition to the original words, I'd like to emphasize one thing (The bolded sentence):
> ..., this setting is ignored. **All the settings on the server ("create") Node will be applied to agent ("join") Nodes automatically.**

@nunix please also help check on this.

Reviewer 2: @johnliu55tw we don't emphasize the difference between server or agent nodes a lot for normal users, so prefer to use the `create`  or `join` mode only.

Author: Good idea, I'll rephrase it a bit.

#### Example

> The configuration below will automatically scan and add storage devices which match the glob
> pattern `/dev/sd*` on every Node.

Reviewer: ```suggestion
The configuration below will automatically scan and add storage devices which will match the glob
pattern `/dev/sd*` on every Node.
```

!!! warning

>     All the data in these storage devices will be destroyed. Use it at your own risk.

Reviewer: ```suggestion
    All the data in these storage devices will be destroyed. Use at your own risk.
```
